lib/src/base-styles/_nx-tile.scss

```
@@ -93,30 +60,28 @@ $nx-tile-background-color: $nx-white;
 // Tile headers
 
 .nx-tile-header {
+  align-items: center;
+  display: grid;
+  display: -ms-grid;
+  grid-template-columns: 1fr auto;
+  -ms-grid-columns: 1fr auto;
   min-height: $nx-tile-header-height;
-  margin: 8px 0 15px 0;
+  margin-bottom: $nx-spacing-md;
   overflow: hidden;
 }
 
 .nx-tile-header__title {
-  display: inline-block;
+  @extend .nx-truncate-ellipsis;
+  grid-column-start: 1;
+  grid-column-end: 1;
+  grid-row-start: 1;
+  grid-row-end: 1;
+  -ms-grid-row: 1;
+  -ms-grid-column: 1;
 
   h1, h2 {
-    display: inline-block;
+    @extend .nx-truncate-ellipsis;
```

Don't we usually use the mixin for this?

```suggestion
    @include truncate-ellipsis;
```